usr/lib/anon-shared-helper-scripts/terminal-wrapper

```diff
@@ -25,11 +27,7 @@ if command -v qubesdb-read >/dev/null 2>&1; then
    if [ "$terminal_emulator_app" = "konsole" ]; then
       terminal_emulator_extra_args="--hold -e"
    fi
-else
-   ## Non-Qubes.
-   if [ "$terminal_emulator_app" = "konsole" ]; then
-      terminal_emulator_extra_args="--hold -e --fullscreen"
-   fi
 fi
 
-$terminal_emulator_app $terminal_emulator_extra_args "$@ ; bash"
+
+$terminal_emulator_app $terminal_emulator_extra_args "bash"
```

Comment:
Pretty sure we need the `$@`. Otherwise no command can be passed and run. Will re-add.